run_tests.sh

```diff
@@ -43,6 +43,7 @@ export CGO_ENABLED=0
 go build ./...
 go build -tags harness -o /dev/null ./client/cmd/simnet-trade-tests
 go build -tags harness -o /dev/null ./client/cmd/mmbot
+go build -tags systray -o /dev/null ./client/cmd/dexc
```

Comment:
Maybe lines for dexc-desktop too

Reply:
dexc-desktop requires cgo. We're explicitly running these builds with `CGO_ENABLED=0`. Can we use cgo in these tests?

Reply:
`CGO_ENABLED=1 go build -tags ...`